spec/support/controller_spec_helper.rb

```diff
@@ -7,16 +7,6 @@ def assigns(key = nil)
     end
   end
 
-  def set_user_privileges(user = FactoryGirl.create(:user_with_group))
-    allow(User).to receive(:server_timezone).and_return("UTC")
-    allow_any_instance_of(described_class).to receive(:set_user_time_zone)
-
-    # TODO: remove these stubs
-    allow(controller).to receive(:check_privileges).and_return(true)
-    login_as user
-    allow_any_instance_of(User).to receive(:role_allows?).and_return(true)
```

Comment:
The new `stub_user`, on the other hand, allows you to create an `(features: :all)` "permitted" or "denied"`(features: :none)` user.  This accounts for the majority of test cases.  When you want to test specific features, you should use `login_as user_with_feature(...)`